en/08_Changelogs/5.4.0.md

```diff
@@ -15,16 +15,57 @@ title: 5.4.0 (unreleased)
+- A new [`CoreKernel::setBootDatabase()`](api:SilverStripe\Core\CoreKernel::setBootDatabase()) method has been added to replace the now deprecated [`DatabaselessKernel`](api:SilverStripe\Core\DatabaselessKernel) class.
+- Two new methods have been added to [`ArrayLib`](api:SilverStripe\ORM\ArrayLib):
+  - [`ArrayLib::insertBefore()`](api:SilverStripe\ORM\ArrayLib::insertBefore()) for inserting a value before another value in an array
+  - [`ArrayLib::insertAfter()`](api:SilverStripe\ORM\ArrayLib::insertAfter()) for inserting a value after another value in an array
+- A new [`DBDatetime::getTimeBetween()`](api:SilverStripe\ORM\FieldType\DBDatetime::getTimeBetween()) method has been added. This method returns the amount of time that has passed between two `DBDateTime` objects as a human-readable string.
+- A new [`AbstractQueuedJob::getQueue()`](api:Symbiote\QueuedJobs\Services\AbstractQueuedJob::getQueue()) static method has been added to get the correct queue constant from a given string or int.
 
 ## API changes
 
 - The class names for the `TopPage` feature in [`dnadesign/silverstripe-elemental`](https://github.com/silverstripe/silverstripe-elemental) do not follow the correct naming convention for Silverstripe CMS. The existing classes have been deprecated and will be renamed to match the correct naming convention in a future major release.
+- [`CliBypass`](api:SilverStripe\Control\Middleware\ConfirmationMiddleware\CliBypass) has been deprecated. It will be removed without equivalent functionality to replace it.
+- [`CliController`](api:SilverStripe\Control\CliController) has been deprecated. It will be replaced with `symfony/console` commands.
+- [`DatabaselessKernel`](api:SilverStripe\Core\DatabaselessKernel) has been deprecated. Use [`CoreKernel::setBootDatabase()`](api:SilverStripe\Core\CoreKernel::setBootDatabase()) instead.
+- [`BuildTask.segment`](api:SilverStripe\Dev\BuildTask->segment) has been deprecated. It will be replaced with a new `$commandName` property.
+- [`BuildTask->description`](api:SilverStripe\Dev\BuildTask->description) has been deprecated. It will be replaced with a new static property with the same name.
+- [`BuildTask::getDescription()`](api:SilverStripe\Dev\BuildTask::getDescription()) has been deprecated. It will be replaced with a new static method with the same name.
+- [`DevBuildController`](api:SilverStripe\Dev\DevBuildController) has been deprecated. It will be replaced with a new `SilverStripe\Dev\Command\DbBuild` class.
+- [`DevConfigController`](api:SilverStripe\Dev\DevConfigController) has been deprecated. It will be replaced with a new `SilverStripe\Dev\Command\ConfigDump` class.
+- [`DatabaseAdminDatabaseAdmin`](api:SilverStripe\ORM\DatabaseAdminDatabaseAdmin) has been deprecated. It will be replaced with a new `SilverStripe\Dev\Command\DbBuild` class.
```

Comment:
I think this is supposed to simply be `DatabaseAdmin`

Reply:
yep good catch. Fixed.